From efca2086ce71ef7a3356a0f5386145a5ce7a98f1 Mon Sep 17 00:00:00 2001
From: jagaur <jagaur@habana.ai>
Date: Mon, 02 Jun 2025 13:27:36 +0300
Subject: [PATCH] [SW-230770] gaudi3: print file and size

This will print the file and its size to check for
any alignment issues seen during some CI run.

Change-Id: I9300a5b855c29ddbfc0553648078694ec39ad6ea
---

diff --git a/board/habanalabs/common/post-image-spi-arc_global.sh b/board/habanalabs/common/post-image-spi-arc_global.sh
index a0b2792..d5726b9 100755
--- a/board/habanalabs/common/post-image-spi-arc_global.sh
+++ b/board/habanalabs/common/post-image-spi-arc_global.sh
@@ -130,7 +130,8 @@
 	FILE=$1
 	OUT_FILE=$2
 	WORD_SIZE=${3}
-
+	local filesize=$(wc -c < $FILE)
+	echo $FILE $filesize $WORD_SIZE
 	${OBJCOPY}  -Ibinary --reverse-bytes=${WORD_SIZE} ${FILE} ${OUT_FILE}
 }
 
